scripts/build.py

```diff
@@ -94,7 +94,7 @@ def get_tag_suffix(self):
 def get_target_platforms():
     return BASE_PLATFORMS + build_private.get_target_platforms()
 
-PACKAGES_ALL="protobuf-3.20.1 waf-2.0.3 junit-4.6 jsign-4.2 protobuf-java-3.20.1 openal-1.1 maven-3.0.1 ant-1.9.3 vecmath vpx-1.7.0 luajit-2.1.0-6c4826f tremolo-b0cb4d1 defold-robot-0.7.0 bullet-2.77 libunwind-395b27b68c5453222378bc5fe4dab4c6db89816a jctest-0.10.2 vulkan-1.3.261.1".split()
```

Comment:
And removed, but I commit Gradle into each folder separately as we talked about it. Later we may decide to install it as well as we installed ant, but then IDE will install it's own version separately in each folder